include/ar71xx-generic.mk

```diff
@@ -84,3 +90,6 @@ WDR4300: \
 	lede-$(LEDE_VERSION)-ar71xx-generic-tl-wdr4300-v1-squashfs-factory.bin \
 	lede-$(LEDE_VERSION)-ar71xx-generic-tl-wdr4300-v1-il-squashfs-factory.bin
 
+WR902AC: \
+	lede-$(LEDE_VERSION)-ar71xx-generic-tl-wr902ac-v1-squashfs-factory.bin \
+	lede-$(LEDE_VERSION)-ar71xx-generic-tl-wr902ac-v1-squashfs-sysupgrade.bin
```

Comment:
Q/RFC:
Is there a particular reason why we need the sysupgrade.bin in that case and not only the -factory.bin ? We only perform complete reflashes of devices.

Reply:
Yeah we dont need that i'll remove it